JumpRoyale/src/TimerOverlay.cs

```diff
@@ -0,0 +1,62 @@
+using System.Threading.Tasks;
+using Godot;
+
+public partial class TimerOverlay : VFlowContainer
+{
+    private const string SpriteName = "Sprite2D";
+    private const string TimerNodeName = SpriteName + "/Timer";
+
+    private static readonly int GameLength = GameOverlay.GameLength;
+
+    private int _timerSeconds = GameLength;
+
```

Comment:
I think we could extract the colors for `UpdateTimer()` here

```suggestion
    private Color _white = new(1, 1, 1, 1);
    private Color _red = new(1, 0, 0, 1);
```